packages/react-composites/src/composites/CallComposite/utils/MediaGalleryUtils.ts

```diff
@@ -190,5 +190,7 @@ export const showDtmfDialer = (
       }
     });
   }
+  /* @conditional-compile-remove(dtmf-dialer-on-by-default) */
+  showDtmfDialer = true;
```

Comment:
the logic is that we are allowing all "identifiers" and so we are just returning true if this is on?